src/main/java/com/monitorjbl/xlsx/impl/StreamingSheetReader.java

```diff
@@ -165,8 +183,10 @@ private void handleEvent(XMLEvent event) throws SAXException {
         currentCell.setContents(formattedContents());
       } else if("row".equals(tagLocalName) && currentRow != null) {
         rowCache.add(currentRow);
+        currentRowNum++;
       } else if("c".equals(tagLocalName)) {
         currentRow.getCellMap().put(currentCell.getColumnIndex(), currentCell);
+        currentColNum++;
```

Comment:
Ah, okay. Never seen this case before, but you've got a unit test so it looks good to me.